templates/nonOverlappingModules.html

```diff
@@ -0,0 +1,52 @@
+$def with (lst_of_independ_mods, lst_of_ay_sem, ay_sem)
+
+$var title:Non-Overlapping Modules
+
+<div class="container container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="row"><!--header-->
+                <h1 class="text-center"><b>Non-Overlapping Modules for $ay_sem</b></h1>
```

Comment:
added `glyphicon-info-sign` to the end of description and rebased tooltip there.
![image](https://cloud.githubusercontent.com/assets/12403982/23910368/6d655324-0914-11e7-8f01-17731f91f3ba.png)